client/starwhale/api/_impl/dataset/loader.py

```diff
@@ -100,8 +140,23 @@ def _do_iter_row(self) -> t.Generator[TabularDatasetRow, None, None]:
                 if rt is None:
                     break
 
-                for row in self.tabular_dataset.scan(rt[0], rt[1]):
-                    yield row
+                if self.dataset_uri.instance_type == InstanceType.CLOUD:
+                    for btch in self.tabular_dataset.scan_btch(
```

Comment:
`for btch in self.tabular_dataset.scan_btch` -> `for rows in self.tabular_dataset.scan_btch` ?

replace `btch` with `rows`?

Reply:
done